consensus/module.go

```diff
@@ -66,7 +66,8 @@ type consensusModule struct {
 	paceMaker         pacemaker.Pacemaker
 	leaderElectionMod leader_election.LeaderElectionModule
 
-	// DEPRECATE: Remove later when we build a shared/proper/injected logger
+	// Internal Logging
```

Comment:
```suggestion
```

Reply:
I don't think this comment adds much. It's clear that `logger` is used for logging within the confines of this module/struct.

Ditto elsewhere.

Reply:
I don't think this comment adds much. It's clear that logger is used for logging within the confines of this module/struct.

Ditto elsewhere.

Reply:
Seems like in other places there's no such comment.